pkgs/development/interpreters/falcon/default.nix

```diff
@@ -1,14 +1,14 @@
-{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, pcre, zlib, sqlite }:
+{ lib, gccStdenv, fetchFromGitHub, cmake, pkg-config, pcre, zlib, sqlite }:
```

Comment:
@siraben why do you use `gccStdenv` instead of `stdenv`?

Reply:
Otherwise the build fails on darwin with clang.